seq_logger.js

```diff
@@ -101,9 +110,7 @@ class SeqLogger {
 
         this._closed = true;
         this._clearTimer();
-        return this.flush().then(() => {
-            this._httpAgent.destroy();
-        });
+        return this.flush().then(() => {});
```

Comment:
the `.then(() => {})` doesn't do anything you can just:

`return this.flush();`

Reply:
Correct